packages/shared/src/components/post/PostActions.tsx

```diff
@@ -166,14 +172,30 @@ export function PostActions({
             Bookmark
           </QuaternaryButton>
           <QuaternaryButton
-            id="share-post-btn"
+            id="copy-post-btn"
```

Comment:
Do we even need to specify the `id` param here?

Reply:
Required by the component itself (legacy) 😅